### PR TITLE
More etiquette for responding to code reviews

### PR DESCRIPTION
**Pull request by josh11b**
Includes the advice from https://github.com/carbon-language/carbon-lang/pull/2022#issuecomment-1239070023

### CODE REVIEW

**Review by chandlerc — CHANGES_REQUESTED** [review]
Temporarily "requesting changes" here so that I can get a screen shot of the UI after doing so...

File: docs/project/code_review.md [hunk 1]
```diff
@@ -248,6 +248,19 @@ For non-optional comments, this section provides several suggestions on how best
...
+review. With regular pushes, GitHub can show individual deltas, giving
+additional flexibility to the reviewer. We squash pull requests when we merge
+them, so the end result is the same.
+
+It is good to reply to every comment so that the reviewer knows you saw them.
+Best practice is to send the reply to all of the comments at once, after the
+files in the pull request have been updated. If there are a lot of replies, it
+can be helpful to include a message saying whether the pull request is now ready
+for re-review.
+
```
Comment (chandlerc) [inline docs/project/code_review.md]: Maybe also suggest flagging the reviewer to re-review using the "request review" refresh-looking button illustrated here:

GitHub Request Review screen shot

**Review by chandlerc — APPROVED** [review]
Looks good. Suggested an addition below that might be helpful, let me know if you'd like me to look at an update, or send this in a follow-up.

File: docs/project/code_review.md (hunk 1 above)
Comment (josh11b) [reply docs/project/code_review.md]: Added, with some text.

File: docs/project/code_review.md [hunk 2]
```diff
@@ -248,6 +248,21 @@ For non-optional comments, this section provides several suggestions on how best
...
+review. With regular pushes, GitHub can show individual deltas, giving
+additional flexibility to the reviewer. We squash pull requests when we merge
+them, so the end result is the same.
+
+It is good to reply to every comment so that the reviewer knows you saw them.
+Best practice is to send the reply to all of the comments at once, after the
+files in the pull request have been updated. If there are a lot of replies, it
+can be helpful to include a message saying whether the pull request is now ready
+for re-review, or press the "request review" button to the right of the
+reviewer's name:
```
Comment (chandlerc) [inline docs/project/code_review.md]:
```suggestion
for another round of review, or press the "Re-request review" button to the right of the
reviewer's name:
```

File: docs/project/code_review.md [hunk 3]
```diff
@@ -248,6 +248,21 @@ For non-optional comments, this section provides several suggestions on how best
...
+![The request review button on GitHub](https://user-images.githubusercontent.com/711534/189789784-fcf18d1b-137b-48ba-959a-0d05cee36c2d.png)
```
Comment (chandlerc) [inline docs/project/code_review.md]:
```suggestion
![The re-request review button on GitHub](https://user-images.githubusercontent.com/711534/189789784-fcf18d1b-137b-48ba-959a-0d05cee36c2d.png)
```

**Review by chandlerc — APPROVED** [review]
Just tweaking wording as the tool-tip (and screen reader) for the button seems to be "re-request review". Still LGTM, feel free to submit w/ these suggestions and any wording/grammar fixes needed.

**Pull request merged**